docs/UserGuide.md

```diff
@@ -205,6 +211,8 @@ Examples:
 - `run -x GET -u https://api.data.gov.sg/v1/environment/pm25`
 - `run -x GET -u https://api.data.gov.sg/v1/environment/uv-index -d {"some": "data"} -h "key: value"`
 
+Description: Performs a GET request to a valid API URL address, without specifying any prefixes or any other parts of an API endpoint.
```

Comment:
Might help to mention that this is the second format since the previous one said 2 formats.